cmd/ebrelayer/main.go

```diff
@@ -360,18 +370,28 @@ func RunInitWitnessCmd(cmd *cobra.Command, args []string) error {
 func replayEthereumCmd() *cobra.Command {
 	//nolint:lll
 	replayEthereumCmd := &cobra.Command{
-		Use:     "replayEthereum [tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMoniker]",
+		Use:     "replayEthereum  [networkDescriptor][tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMoniker]",
```

Comment:
Instead of adding networkDescriptor this way, can we add it using the new AddRelayerFlagsToCmd function?  We should be using the Cobra features to add options, generally.

Reply:
Do we need to keep replayEthereum()?

Reply:
I agree we can remove replay functions in peggy2, will use a new commit to do it. For flags instead of arguments, I will create a separate pr to refactor all those arguments before.